src/factories/tracking/ActsTrajectoriesMerger_factory.h

```diff
@@ -26,6 +26,10 @@ class ActsTrajectoriesMerger_factory : public JOmniFactory<ActsTrajectoriesMerge
   void ChangeRun(int32_t /* run_number */) {}
 
   void Process(int32_t /* run_number */, uint64_t /* event_number */) {
+
+    // FIXME clear output since it may not have been initialized or reset
+    m_acts_trajectories_output().clear();
```

Comment:
```suggestion
    // FIXME clear output since it may not have been initialized or reset
    // See https://github.com/eic/EICrecon/issues/1961
    m_acts_trajectories_output().clear();
```